requirements.txt

```diff
@@ -1,25 +1,27 @@
 alabaster==0.7.12
+appdirs==1.4.4
 attrs==20.3.0
 Babel==2.9.0
+black==20.8b1
 cachetools==4.2.1
 certifi==2020.12.5
 chardet==4.0.0
 click==7.1.2
 click-log==0.3.2
-commonmark==0.9.1
-coverage==5.4
+coverage==5.5
 decorator==4.4.2
-Deprecated==1.2.11
+Deprecated==1.2.12
 docutils==0.16
 entrypoints==0.3
-google-api-core==1.25.1
```

Comment:
just confirming - not needed?

Reply:
Oh, it's needed. The version was upgraded to `google-api-core==1.26.1`, so you can see it on the right hand side of the diff.

Reply:
ops - I missed that one :) thanks!